ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,503 @@
+	case appColor
+	case neutral
+	case custom
+}
+
+/// Card style can be announcement, xSmallHorizontal, xSmallVertical, small, or medium)
+/// announcement has  a title, a subtitle, and a button that are centrally aligned, and a featured image (the card is not clickable other than the button)
+/// small and medium styles have a title, a subtitle, an icon and a featured image. The whole card is clickable
+/// xSmall styles have a title, an icon and an optional subtitle. The whole card is clickable
+@objc(MSFCardStyle)
+public enum CardStyle: Int, CaseIterable {
+    case xSmallHorizontal
+    case xSmallVertical
+    case small
+    case medium
+    case announcement
+
+    var imageHeight: CGFloat {
+        switch self {
+        case .medium:
+            return 128
+        case .small:
+            return 76
+        case .announcement:
+            return 100
+        case .xSmallHorizontal, .xSmallVertical:
+            return .zero
+        }
+    }
+
+    var iconSize: CGSize {
+        switch self {
+        case .medium, .small:
+            return CGSize(width: 16, height: 16)
+        case .xSmallHorizontal, .xSmallVertical:
+            return CGSize(width: 18, height: 18)
+        case .announcement:
+            return .zero
+        }
+    }
+
+	var textAlignment: NSTextAlignment {
+		switch self {
+		case .announcement:
+			return NSTextAlignment.center
+		default:
+			return NSTextAlignment.left
+		}
+	}
+
+    var cardMinWidth: CGFloat {
+       switch self {
+       case .medium:
+            return 228
+       case .small:
+            return 135
+       case .xSmallHorizontal:
+           return 156
+       case .xSmallVertical:
+            return 120
+       case .announcement:
+           return 334
+       }
+   }
+}
+
+/**
+`CardView` is a UIView used to display information in a card.
+
+A card has a title, a style, a background style,  and the following optional components; subtitle, icon, featured image, and a button.
+Refer to the `CardStyle` documentation for a detailed description of components in each CardView style.
+
+Use `titleNumberOfLines` and `subtitleNumberOfLines`  to set the number of lines the title and subtitle should have respectively. When the string can not fit in the number of lines set, it will get truncated.
+
+Use one of the defined background styles for appColor or a neutral grey color.
+When CardBackgroundStyle.custom is used, a default color will be set unless a background color is provided by setting the `cardBackgroundColor` property.
+
+Conform to the `CardDelegate` in order to provide a handler for tap events. All CardView styles are clickable other than announcement, which has a dedicatred button inside the card.
+*/
+@objc(MSFCardView)
+open class CardView: UIView {
+    /// delegate to handle user interaction with the CardView
+    @objc public weak var delegate: CardDelegate?
+
+	@objc open var style: CardStyle = .medium {
+		didSet {
```

Comment:
Seems like there are indentation issues throughout this file making it harder to review